core/modulesource.go

```diff
@@ -273,7 +276,7 @@ func (src *ModuleSource) ModuleConfig(ctx context.Context) (*modules.ModuleConfi
 type LocalModuleSource struct {
 	RootSubpath string `field:"true" doc:"The path to the root of the module source under the context directory. This directory contains its configuration file. It also contains its source code (possibly as a subdirectory)."`
 
-	ContextDirectory dagql.Instance[*Directory] `field:"true" doc:"The directory containing everything needed to load load and use the module."`
+	ContextDirectory dagql.Nullable[dagql.Instance[*Directory]] `field:"true" doc:"The directory containing everything needed to load load and use the module."`
```

Comment:
@sipsma I had to make this nullable since otherwise it's possible to call it and get back a zero-value `dagql.Instance[*Directory]` which leads to a nil panic. (Nothing load-bearing hit this, I was just monkeying around in the GraphQL playground and accidentally triggered it.)